gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -285,6 +285,16 @@ security:
   # If true create on AM bootstrap an inline identity provider with an admin user (login: admin)
   # this is the legacy mode
   defaultAdmin: true
+  accountAccessTokens:
+    ## how should the tokens be encoded in the database
+    encoder:
+      ##  encoding algorithm:
+      ## - BCrypt: account tokens are hashed with bcrypt (supports only $2a$ algorithm)
+      ## - none: account tokens are not hashed/encrypted
+      algorithm: BCrypt
```

Comment:
**todo**: as discuss on slack, BCrypt will be the only one available no need to make it configurable